### PR TITLE
Added support for the config form notes.

### PR DESCRIPTION
**Pull request by shinde-rahul**
Example PR for https://github.com/mautic/mautic/pull/11451

### CODE REVIEW

**Comment by kuzmany** [comment]
It works for me. Already approved related Mautic PR https://github.com/mautic/mautic/pull/11451#pullrequestreview-1135852180

**Review by kuzmany — DISMISSED** [review]

File: Integration/Support/ConfigSupport.php
Comment (kuzmany) [inline Integration/Support/ConfigSupport.php:7]: Probably we should use more detailed variable name like IntegrationNote for DTO. We've already use Note entity, @escopecz what is your opinion?

Comment (escopecz) [reply Integration/Support/ConfigSupport.php:7]: I think it's fine. We have namespaces for this very purpose

**Review by kuzmany — APPROVED** [review]
Good

**Review by escopecz — APPROVED** [review]
👍 Good to go once the parent PR is merged

**Comment by escopecz** [comment]
This may have to be rebased on mautic-5 branch as the Mautic PR will be released in M5

**Comment by shinde-rahul** [comment]
@escopecz, we have https://github.com/mautic/mautic/pull/11451 for mautic-5. I forgot why I raised it for M4.

**Comment by shinde-rahul** [comment]
Closing this.

**Pull request closed without merging**
